_includes/docs/user-guide/alarms.md

```diff
@@ -1,45 +1,45 @@
 * TOC
 {:toc}
 
-ThingsBoard provides the ability to create and manage alarms related to your entities: devices, assets, customers, etc.
+In the context of ThingsBoard, an "Alarm" refers to a notification system that is triggered when certain conditions occur in your devices, assets, customers, etc. 
+These alarms are configured to monitor various situations such as exceeding threshold values, equipment malfunctions, or other important events requiring attention. 
+Alarms assist in promptly addressing issues, preventing potential negative consequences, and ensuring efficient system operation. 
+
 For example, you may configure ThingsBoard to automatically create an alarm when the temperature sensor reading is above a certain threshold.
 Of course, this is a very simplified case, and real scenarios can be much more complex.
 
-
 ## Main concepts
 
 Let's review the main concepts of the alarm below:
 
-##### Originator
+**Originator**
```

Comment:
I like the old structure. Please revert the changes to the structure. just add more info to each section